### DISCUSSION TITLE
Type with name 'Customer' was not found but I see Type with name 'Customer'

### DISCUSSION THREAD

**Discussion by ZachMurray** [ArcadeData/arcadedb#1095] [Q&A] [answered]
Why does ArcadeDB-Java report not finding `Customer` when I can clearly see `Customer` on the ArcadeDB-Studio? 

![image](https://github.com/ArcadeData/arcadedb/assets/20545037/c20cdccc-8aa6-4127-a169-bd694368fd79)

**Comment by lvca** [comment, marked as answer]
From your code, you're dropping and creating the database. Try to refresh Studio and you should see the Customer type gone. Or try to comment the drop database part and re-run it.

**Reply by ZachMurray** [reply]
Ah, I was wondering if ArcadeDB templated out a generic or automatic make-class-match-on-create.
